contributors/devel/release/patch_release.md

```diff
@@ -0,0 +1,87 @@
+# Building Kubernetes patch releases
+
+This document describes the process for creating Kubernetes patch releases and communicating publishing
+status regarding known issues impacting the current release.
+
+## Process
+
+After a Kubernetes minor release, patch releases are managed by the Kubenretes patch release czar.
+The patch release czar works with SIGs to coordinate merging cherrypicks into the release branch,
+then cutting and publishing a new patch release.
+
+## Communicating fixes to the community
+
+The canonical source of issues in a patch release will be communicated through GitHub issues.
+
+To make it easy for the community to understand the status of each patch release, GitHub issues
+must follow a consistent format and contain basic information about the issue.
+
+GitHub issues targeted at patch releases must contain the following items:
+
+### Release label
+
+The release label contains information about which releases are impacted by the issue, and
+which release the issue is resolved in.
+
+- `release/introduced-in/1.7.0`
+- `release/resolved-in/1.7.2`
```

Comment:
I think we want to filter to be able to ask - which issues were resolved in patch-release-X.  This would allow us to generate the doc you are suggesting.